src/plugins/data/public/ui/index_pattern_select/index_pattern_select.test.tsx

```diff
@@ -49,7 +49,7 @@ describe('IndexPatternSelect', () => {
 
     bulkGetMock.mockResolvedValue({ savedObjects: [{ attributes: { title: 'test1' } }] });
     compInstance.debouncedFetch('');
-    await new Promise((resolve) => setTimeout(resolve, 300));
+    await new Promise((resolve) => setTimeout(resolve, 600));
```

Comment:
Why this change?

Reply:
This test case was a bit flakey in nature. hence to resolve it I had increased the timeout.

Reply:
This is not the right way to deal with timers.